pkl-cli/pkl-cli.gradle.kts

```
@@ -138,9 +138,10 @@ fun Exec.configureExecutable(isEnabled: Boolean, outputFile: Provider<RegularFil
   enabled = isEnabled
   dependsOn(":installGraalVm")
 
-  inputs.files(sourceSets.main.map { it.output })
-  inputs.files(configurations.runtimeClasspath)
+  inputs.files(sourceSets.main.map { it.output }).withPropertyName("mainSourceSets").withPathSensitivity(PathSensitivity.RELATIVE)
+  inputs.files(configurations.runtimeClasspath).withPropertyName("runtimeClasspath").withNormalizer(ClasspathNormalizer::class)
```

We should provide `bin/native-image` as an input here as well, right?

```suggestion
  inputs.files(configurations.runtimeClasspath).withPropertyName("runtimeClasspath").withNormalizer(ClasspathNormalizer::class)
inputs.files(file(buildInfo.graalVm.baseDir).resolve("bin/native-image")).withPropertyName("graalVmNativeImage").withPathSensitivity(PathSensitivity.ABSOLUTE)
```